spec/models/manageiq/providers/cloud_manager/provision_workflow_spec.rb

```diff
@@ -189,9 +188,7 @@
 
     context "#allowed_cloud_networks" do
       it "without a zone", :skip_before do
-        network_manager
-
-        expect(workflow.allowed_cloud_networks.length).to be_zero
+        expect(workflow.allowed_cloud_networks.length).to eq(1)
```

Comment:
Since you're going to ask about this one, @jrafanie, the original setup of this test was wrong, and the factory creates the cloud_network when it creates the ems_amazon on 5, and since it has a cloud_network, the expected behavior of the ```allowed_cloud_networks``` method when given one cloud_network is to return it (because in the dialog for the provisioning workflow logic, if there's a single choice, it'll get picked by default). This should now be doing the right thing and I checked with GM and stuff.

Reply:
🤷‍♂ :shipit: